src/main/java/seedu/address/logic/commands/AddPersonCommand.java

```diff
@@ -72,13 +75,15 @@ public AddPersonCommand(
             Phone phone,
             Email email,
             InternshipId internshipId,
-            Set<Tag> tags) {
-        requireAllNonNull(name, phone, email, tags);
+            Set<Tag> tags,
+            Index linkIndex) {
+        requireAllNonNull(name, phone, email, tags, linkIndex);
```

Comment:
linkIndex is optional so could be null.
```suggestion
        requireAllNonNull(name, phone, email, tags);
```